ynput/OpenPype
# Nuke: removed default workfile template builder preset

## Changelog Description
Default for workfile template builder should have been empty.

## Testing notes:
1. open Studio project settings at `project_settings/nuke/templated_workfile_build/profiles`
2. Notice that ther is no preset (if any override then remove those)
3. open Studio project settings at `project_settings/nuke/workfile_builder/profiles`
4. Notice that ther is no preset (if any override then remove those)

## Screenshot
![image](https://user-images.githubusercontent.com/40640033/231495711-a30b821a-d6ee-4d50-a8bc-8cf9fb2927e2.png)
